Userland/Services/WebContent/PageClient.h

```diff
@@ -151,6 +155,10 @@ class PageClient final : public Web::PageClient {
     Web::CSS::PreferredColorScheme m_preferred_color_scheme { Web::CSS::PreferredColorScheme::Auto };
 
     RefPtr<WebDriverConnection> m_webdriver;
+
+#ifdef HAS_ACCELERATED_GRAPHICS
+    OwnPtr<AccelGfx::Context> m_gl_context;
```

Comment:
Maybe we should be forward-looking and not call this "GL context"? :^)

Reply:
indeed. I changed it to `m_accelerated_graphics_context`.